frontend/apps/web/components/ui/dialog.tsx

```diff
@@ -12,14 +12,16 @@ const DialogTrigger = DialogPrimitive.Trigger;
 
 const DialogPortal = DialogPrimitive.Portal;
 
+const DialogClose = DialogPrimitive.Close;
+
 const DialogOverlay = React.forwardRef<
   React.ElementRef<typeof DialogPrimitive.Overlay>,
   React.ComponentPropsWithoutRef<typeof DialogPrimitive.Overlay>
 >(({ className, ...props }, ref) => (
   <DialogPrimitive.Overlay
     ref={ref}
     className={cn(
-      'fixed inset-0 z-50 bg-background/80 backdrop-blur-sm data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0',
+      'fixed inset-0 z-50 bg-background/80 dark:bg-black/40 backdrop-blur-sm data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0',
```

Comment:
This was overwritten by shadcn to include `bg-black/80` in place of the `bg-background/80` we have here.
It also removed the `backdrop-blur-sm`. However I think the blur looks nice so I added it back in.

I added a new dark mode styling because `bg-background/80` made it a little difficult to see where the dialog started and ended in dark mode. Having the `dark:bg-black/40` seems like a nice distinction for dark mode.